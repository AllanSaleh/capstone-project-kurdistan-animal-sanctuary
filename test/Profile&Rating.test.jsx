import React from "react";
import renderer from "react-test-renderer";
import { test, expect } from "@jest/globals";
import ProfileandRating from "../components/Profile&Rating";

test("proparly renders the Profile&Rating component", () => {
  const tree = renderer.create(<ProfileandRating />).toJSON();
  expect(tree).toMatchSnapshot();
});
