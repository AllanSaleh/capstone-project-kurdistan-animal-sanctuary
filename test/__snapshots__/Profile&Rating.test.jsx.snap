// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`proparly renders the Profile&Rating component 1`] = `
<div
  className="flex flex-wrap md:flex-nowrap  py-14  font-Quicksand w-full"
>
  <div
    className="flex flex-row "
  >
    <div
      className="w-40 md:w-72 h-40 md:h-72 rounded-full "
    >
      <img
        alt="animal profile "
        className=" w-36 md:w-60 h-36 md:h-60 border-2 border-secondary p-2 rounded-full"
      />
    </div>
    <div
      className="w-48"
    >
      <div
        className="px-5 py-4 font-bold text-primary text-xl md:text-2xl "
      >
        <div />
      </div>
      <div
        className="hidden md:inline-block w-52 md:w-72 h-8 p-5 pt-6"
      >
        <div
          className="react-stars-wrapper-"
          style={
            Object {
              "display": "flex",
            }
          }
        >
          <div
            aria-label="add rating by typing an integer from 0 to 5 or pressing arrow keys"
            className=""
            onKeyDown={[Function]}
            style={
              Object {
                "overflow": "hidden",
                "position": "relative",
              }
            }
            tabIndex={0}
          >
            <p
              role="status"
              style={
                Object {
                  "left": "-200rem",
                  "position": "absolute",
                }
              }
            >
              0
            </p>
          </div>
        </div>
      </div>
      <div
        className=" inline-block md:hidden w-48 md:w-72 h-8 p-5 pt-6"
      >
        <div
          className="react-stars-wrapper-"
          style={
            Object {
              "display": "flex",
            }
          }
        >
          <div
            aria-label="add rating by typing an integer from 0 to 5 or pressing arrow keys"
            className=""
            onKeyDown={[Function]}
            style={
              Object {
                "overflow": "hidden",
                "position": "relative",
              }
            }
            tabIndex={0}
          >
            <p
              role="status"
              style={
                Object {
                  "left": "-200rem",
                  "position": "absolute",
                }
              }
            >
              0
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="grid grid-flow-col w-4/5  "
  >
    <div
      className="justify-self-start pt-8 md:pt-0 items-start md:justify-self-end md:pt-12"
    >
      <button
        className="mr-3 w-28 h-10 text-primary bg-highlight hover:bg-darken rounded-lg "
        type="button"
      >
        Message
      </button>
    </div>
  </div>
</div>
`;
